features/identity-event/org.wso2.carbon.identity.event.server.feature/resources/org.wso2.carbon.identity.event.server.feature.default.json

```diff
@@ -170,7 +170,8 @@
   "identity_mgt.events.schemes.identityOauthEventHandler.module_index": "28",
   "identity_mgt.events.schemes.identityOauthEventHandler.subscriptions": [
     "POST_SET_USER_CLAIMS",
-    "POST_SET_USER_CLAIM"
+    "POST_SET_USER_CLAIM",
+    "POST_UPDATE_USER_LIST_OF_ROLE"
```

Comment:
we will have to revoke tokens/sessions on delete role, permission update as well.

Reply:
Addressed with commit f0f156e8e8990df3af44922e65acf1061762fda5